### PR TITLE
Add a case study about BOE for unionbigdata

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add case study `数之联在 BOE 的 I3Plat 建设中使用 Karmada`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

Deal with the translation of Chinese to English documents later.

### CODE REVIEW

File: docs/casestudies/unionbigdata.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+## 行业背景
+
+在液晶面板生产领域，由于多种因素，产品常出现不良品。为此，关键工艺节点后引入了自动光学检测（AOI）设备，通过光学原理检测常见缺陷。然而，现有 AOI 设备仅识别缺陷有无，需要人工分类和识
+别假缺陷，这一过程耗时且影响生产效率。[BOE 集团](https://www.boe.com/)引入自动缺陷分类系统（ADC）以提高判定准确性并减轻劳动强度，使用深度学习技术自动分类 AOI
+输出的缺陷图片，并筛除误判，从而提高生产效率。
+
+BOE 率先在一个工厂引入 ADC，后续在其他工厂推广，节省人力资源，提高判定效率。尽管如此，由于工艺复杂和供应商差异，现场建设呈现出割裂和分散管理的趋势，给数据共享和运维带来困难。为解
+决这些问题，BOE 启动了工业智能检测平台（I3Plat）的建设，该平台利用人工智能技术，标准化智能检测并提高生产效率和良率。
+
+![](./static/unionbigdata_01.png)
```
Comment (windsonsea) [inline docs/casestudies/unionbigdata.md:14]:
```suggestion
![工业智能检测平台](./static/unionbigdata_01.png)
```
这种 alt text 还是要加一下，在网络很差且图片无法加载时，可以给读者一些提示

**Comment by XiShanYongYe-Chang** [comment]
Hi @pandaoknight, can you help take a look?

File: docs/casestudies/unionbigdata.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+```
+/apis/cluster.karmada.io/v1alpha1/clusters/%s/proxy/api/v1/namespaces/%s/services/%s/proxy/%s
+```
+
+#### 1、集群监控
+
+针对在线的集群，中心云系统可对内存、CPU、磁盘、网络流入流出速率、GPU、日志等指标进行监控数据展示，并可切换集群进行数据查看。
+
+![](./static/unionbigdata_04.png)
```
Comment (windsonsea) [inline docs/casestudies/unionbigdata.md:56]: ditto，不要偷懒 😸

File: docs/casestudies/unionbigdata.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply docs/casestudies/unionbigdata.md:14]: 感谢~

File: docs/casestudies/unionbigdata.md (hunk 2 above)
Comment (XiShanYongYe-Chang) [reply docs/casestudies/unionbigdata.md]: 已修改，请帮忙再检视下~

**Review by windsonsea — COMMENTED** [review]
这个 folder 后续要改成英文版内容吧：

- [docs/casestudies/unionbigdata.md](https://github.com/karmada-io/website/pull/537/files#diff-597f9f430ac50e9cdd38036fd8cce725f3448bbb01b49e1a287e6201722d6754)

**Comment by XiShanYongYe-Chang** [comment]
> 这个 folder 后续要改成英文版内容吧：

是的，但图片的部分不容易修改，关于其中一些平台的截图可能也无法进行修改。

**Comment by pandaoknight** [comment]
@XiShanYongYe-Chang @windsonsea 
I will provide the English version of the document next week and will redraw the images within it, with the text in the images changed to English. 

By the way, please take a look at https://github.com/karmada-io/website/pull/531,
please change the company name from DSK to Unionbigdata. 

Thanks.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**